pkg/splunk/enterprise/util.go

```diff
@@ -69,6 +69,86 @@ func getIndexerExtraEnv(cr splcommon.MetaObject, replicas int32) []corev1.EnvVar
 	}
 }
 
+// getClusterMasterExtraEnv returns extra environment variables used by indexer clusters
+func getClusterMasterExtraEnv(cr splcommon.MetaObject, spec *enterprisev1.CommonSplunkSpec) []corev1.EnvVar {
```

Comment:
Seems like we should have a single API getExtraEnv(with possible wrappers in CR specific files such as getClusterMasterExtraEnv etc.. ) i.e

getExtraEnv(cr splcommon.MetaObject, spec *enterprisev1.CommonSplunkSpec, int32 replicas) []corev1.EnvVar)

Maybe this can be taken up in the future.